src/BayesianOptimization.jl

```diff
@@ -164,5 +232,80 @@ function _evaluate_function(o, x)
     end
     return y
 end
+"""
+    optimize(f, lowerbounds, upperbounds; <keyword arguments>)
+
+Find an optimizer between `lowerbounds` and `upperbounds` of a function `f` using default parameters.
+
+Default parameters can be overwritten by passing respective values as keyword arguments.
+
+# Arguments
+
+TODO: document defaults, keywords: 
+model, acquisition, modeloptimizer, sense, maxiterations, maxduration, acquisitionoptions, repetitions, verbosity, initializer_iterations, initializer
+"""
+function optimize(f, lowerbounds, upperbounds; optkwargs...)
+    args, kwargs = merge_with_defaults(f, lowerbounds, upperbounds, optkwargs)
+    opt = BOpt(args...; kwargs...)
+    boptimize!(opt)
+end
+"""
+    merge_with_defaults(f, lowerbounds, upperbounds, optkwargs)
+"""
+function merge_with_defaults(f, lowerbounds, upperbounds, optkwargs)
+    # the same order of args as in BOpt constructor
+    args_keys = [:model, :acquisition, :modeloptimizer]
```

Comment:
Use tuples `()` instead of vectors `[]` (they don't allocate at runtime).
Also for `kwargs_keys`.

Reply:
oki